packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed a bug where uploaded special characters may be corrupted if they fall on chunk borders for streaming. [#2556](https://github.com/zowe/zowe-cli/issues/2555)
```

Comment:
I would like to have Ana's opinion on this one 😋 

What do you think about any of these? 😋 
- BugFix: Resolved an issue where streaming uploads of special characters could result in data corruption at chunk boundaries
- BugFix: Resolved an issue where uploaded special characters may be corrupted if they fall at data transfer segments during streaming